wallets/service/channel_transaction_status_service.py

```diff
@@ -0,0 +1,26 @@
+from common.blockchain_util import BlockChainUtil
+from common.constant import TransactionStatus
+from wallets.dao.channel_transaction_status_data_access_object import \
+    ChannelTransactionStatusDataAccessObject
+from wallets.config import NETWORKS, NETWORK_ID
+
+
+class ChannelTransactionStatusService:
+    def __init__(self, repo, net_id):
+        self.net_id = net_id
+        self.repo = repo
+        self.obj_blockchain_util = BlockChainUtil(provider_type="HTTP_PROVIDER",
+                                                  provider=NETWORKS[NETWORK_ID]['http_provider'])
+        self.obj_channel_transaction_history_dao = ChannelTransactionStatusDataAccessObject(repo=self.repo)
+
```

Comment:
can it be ChannelService , and functions like open_Channel etc from wallet be moved here?